src/_includes/head.html

```diff
@@ -2,26 +2,22 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-
   <title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
   <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
-
-  <link rel="stylesheet" href="{{ "/styles/site.css" | prepend: site.baseurl }}">
-  <link rel="stylesheet" href="{{ "/css/theme.css" | prepend: site.baseurl }}">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400" rel="stylesheet">
+  <link rel="stylesheet" href="{{ "/css/site.css" | prepend: site.baseurl }}">
```

Comment:
is there a difference between this style and `{{ site.baseurl }}/css/site.css` like it is in other places? If not, let's pick one and be consistent about it.

```
<li><a href="{{ site.baseurl }}/get-started/beam-overview/">Beam Overview</a></li>
```